packages/client/components/DiscussionThreadListEmptyTranscriptState.tsx

```diff
@@ -0,0 +1,156 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '~/styles/paletteV3'
+import makeMinWidthMediaQuery from '~/utils/makeMinWidthMediaQuery'
+import useAtmosphere from '../hooks/useAtmosphere'
+import useForm from '../hooks/useForm'
+import useMutationProps from '../hooks/useMutationProps'
+import AddTranscriptionBot from '../mutations/AddTranscriptionBotMutation'
+import linkify from '../utils/linkify'
+import Legitity from '../validation/Legitity'
+import FlatButton from './FlatButton'
+import StyledError from './StyledError'
+import {DiscussionThreadListEmptyTranscriptState_meeting$key} from '~/__generated__/DiscussionThreadListEmptyTranscriptState_meeting.graphql'
+import {ZoomSVG} from './ZoomSVG'
+
+const mobileBreakpoint = makeMinWidthMediaQuery(380)
+
+const DiscussionThreadEmptyStateRoot = styled('div')({
+  padding: '12px 24px',
+  margin: 'auto',
+  display: 'flex',
+  flexDirection: 'column',
+  minHeight: 0
+})
+
+const EmptyDiscussionContainer = styled('div')({
+  width: 160,
+  margin: '14px auto',
+  textAlign: 'center',
+  [mobileBreakpoint]: {
+    width: 260
+  }
+})
+
+const Message = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 14,
+  textAlign: 'center',
+  lineHeight: '20px',
+  margin: '24 0'
+})
+
+const StyledButton = styled(FlatButton)({
+  background: PALETTE.SKY_500,
+  color: PALETTE.WHITE,
+  fontWeight: 600,
+  minWidth: 36,
+  marginTop: 24,
+  width: '50%',
+  marginBottom: 8,
+  ':hover': {
+    backgroundColor: PALETTE.SKY_600
+  }
+})
+
+const Wrapper = styled('div')({
+  alignItems: 'center',
+  display: 'flex',
+  justifyContent: 'center',
+  flexWrap: 'wrap',
+  flex: 1
+})
```

Comment:
+1 I don't like that we're introducing more styled components when we're actively trying to get rid of it.